styles.css

```diff
@@ -1,7 +1,73 @@
-h1 {
-  color: red;
+.flex-container {
+  padding: 0;
+  margin: 0;
+  text-decoration: none;
+  list-style: none;
+  box-sizing: border-box;
+  font-family: Arial, Helvetica, sans-serif;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
 }
 
-p {
-  color: blue;
+nav {
+  height: 80px;
+  width: 100%;
+  font-size: 30px;
+}
+
+nav label {
+  float: right;
+  line-height: 80px;
+  position: relative;
+  top: 50px;
+  right: 70px;
+}
+
+.logo {
+  position: relative;
+  float: left;
+  top: 70px;
+  left: 20px;
+  font-weight: bold;
+}
+
+section h1 {
+  color: #172b4d;
+  text-align: center;
+  margin: 30px;
+  margin-top: 140px;
+  font-size: 40px;
+  font-weight: bold;
+}
+
+section h2 {
+  text-align: center;
+  margin: 30px;
+  font-size: 40px;
+  font-weight: bold;
+  color: #36b37e;
+}
+
+section p {
+  color: #172b4d;
+  text-align: center;
+  text-indent: 28px;
+  font-size: 24px;
+  font-weight: bold;
+  font-family: 'inter';
+  font-style: normal;
+  line-height: 28px;
+  margin: 30px;
+  background-color: transparent;
+}
+
+.icon i {
+  padding: 10px;
+  text-align: center;
+  font-size: 34px;
+}
+
+.icon {
+  text-align: center;
```

Comment:
- [ ] Great job. You were able to add the social media icons. However, their colors are not matching the ones on Figma. Some of them are not even matching the ones on Figma but its ok you are so close 🏃‍♂️ 
Look your design
![image](https://user-images.githubusercontent.com/90222110/176924556-e76d35d6-f663-4b41-bc5b-1169219246c7.png)
Figma design
![image](https://user-images.githubusercontent.com/90222110/176924983-b60229aa-d8e4-4aa9-ba80-7368a04bcf50.png)
Please add the right color in order to meet what is on the design given.